src/printer/layers.js

```diff
@@ -267,3 +277,120 @@ function createLayerWMTS(layerSpec, rootFrameState) {
     layerSpec.opacity
   );
 }
+
+/**
+ * @param {WfsLayer} layerSpec
+ * @param {FrameState} rootFrameState
+ * @return {Observable<LayerPrintStatus>}
+ */
+function createLayerWFS(layerSpec, rootFrameState) {
+  const width = rootFrameState.size[0];
+  const height = rootFrameState.size[1];
+  const context = createCanvasContext2D(width, height);
+  context.canvas.style = {};
+  let frameState;
+  let renderer;
+  let format;
+  let gmlFormats = {
+    '1.0.0': new GML2(),
+    '1.1.0': new GML3(),
+    '2.0.0': new GML32(),
+  };
+
+  if (layerSpec.format === 'gml') {
+    format = new WFS({
+      version: layerSpec.version,
+      gmlFormat: gmlFormats[layerSpec.version],
+    });
+  } else {
+    format = new GeoJSON();
+  }
+
+  let vectorSource = new VectorSource({
+    format: format,
+    loader: function (extent, resolution, projection) {
+      let proj = projection.getCode();
+      let xhr = new XMLHttpRequest();
+      let params = {
+        service: 'WFS',
+        request: 'GetFeature',
+        version: layerSpec.version ? layerSpec.version : '1.1.0',
+        typename: layerSpec.layer,
```

Comment:
not the same name for WFS 2.0.0

Reply:
Ah that's right, thanks. Somehow WFS 2.0.0 still doesn't work. When I try to open the `GetFeature` url in the browser it downloads the response for 2.0.0. Why? Could this be the reason?